packages/oid4vci-rest-client/plugin.schema.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
this file should be omitted and be build by the generate-plugin-schema script. if there are non serializable objects then you should provide it.

Reply:
removed